test/AElf.OS.Network.Grpc.Tests/Helpers/GrpcTestPeerHelper.cs

```diff
@@ -17,23 +19,26 @@ public static GrpcPeer CreateBasicPeer(string ip, string pubkey)
     {
         return CreatePeerWithInfo(ip,
             new PeerConnectionInfo
-                { Pubkey = pubkey, SessionId = new byte[] { 0, 1, 2 }, ConnectionTime = TimestampHelper.GetUtcNow() });
+            {

```

Comment:
It is highly recommended to write only one attribute assignment per line. Of course, it is acceptable to follow the original code pattern that placed all of assignments in a line, but it is not acceptable to place 3 in one line, and another in another line at the same time.